src/ppx/grfx/dx11/dx11_command.cpp

```diff
@@ -464,6 +464,16 @@ void CommandBuffer::CopyBufferToBuffer(
     mCommandList.CopyBufferToBuffer(&copyArgs);
 }
 
+void CommandBuffer::CopyBufferToImage(
+    const std::vector<grfx::BufferToImageCopyInfo>& pCopyInfos,
+    grfx::Buffer*                                   pSrcBuffer,
+    grfx::Image*                                    pDstImage)
+{
+    for (auto& pCopyInfo : pCopyInfos) {
+        CopyBufferToImage(&pCopyInfo, pSrcBuffer, pDstImage);
+    }
+}
+
 void CommandBuffer::CopyBufferToImage(
     const grfx::BufferToImageCopyInfo* pCopyInfo,
```

Comment:
shall this be replaced with a const-ref? (now that you provide a const-ref vector api?)